docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Paddle.md

```diff
@@ -2,18 +2,123 @@
+    # create a paddle.hapi.model.Model format model
+    model = paddle.Model(paddle.vision.models.resnet50(pretrained=False, num_classes=10))
+    # prepare dataset
+    train_dataset = paddle.vision.datasets.Cifar10(mode='train')
+    val_dataset = paddle.vision.datasets.Cifar10(mode='test')
+    # define optimizer
+    optimizer = paddle.optimizer.Momentum(learning_rate=0.01,
+                                          momentum=0.9,
+                                          weight_decay=paddle.regularizer.L2Decay(1e-4),
+                                          parameters=model.parameters())
+    model.prepare(optimizer, paddle.nn.CrossEntropy(), paddle.metric.Accuracy(topk=(1, 5)))
+    # start training
+    model.fit(train_dataset,
+              val_dataset,
+              epochs=50,
+              batch_size=64,
+              save_dir="./output",
+              num_workers=8)
+    # convert to OpenVINO IR format
+    ov_model = convert_model(model)
+
+    # optional: serialize OpenVINO IR to *.xml & *.bin
+    from openvino.runtime import serialize
+    serialize(ov_model, "ov_model.xml", "ov_model.bin")
+
+* ``paddle.fluid.dygraph.layers.Layer``
+
+  ``example_input`` is required while ``example_output`` is optional, which accept the following formats:
+
+  ``list`` with tensor(``paddle.Tensor``) or InputSpec(``paddle.static.input.InputSpec``)
+
+  Example of converting ``paddle.fluid.dygraph.layers.Layer`` format model:
+
+  .. code-block:: python
+  
+  	import paddle
+  	from openvino.tools.mo import convert_model
+  
+    # create a paddle.fluid.dygraph.layers.Layer format model
+  	model = paddle.vision.models.resnet50()
+  	x = paddle.rand([1,3,224,224])
+
+    # convert to OpenVINO IR format
+  	ov_model = convert_model(model, example_input=[x])
+
+    # optional: serialize OpenVINO IR to *.xml & *.bin
+    from openvino.runtime import serialize
+    serialize(ov_model, "ov_model.xml", "ov_model.bin")
+  
+* ``paddle.fluid.executor.Executor``
+
+  ``example_input`` and ``example_output`` are required, which accept the following formats:
+
+  ``list`` or ``tuple`` with variable(``paddle.static.data``)
+
+  Example of converting ``paddle.fluid.executor.Executor`` format model:
+
+  .. code-block:: python
+
+    import paddle
+  	from openvino.tools.mo import convert_model
+
+    paddle.enable_static()
+
+    # create a paddle.fluid.executor.Executor format model
+    x = paddle.static.data(name="x", shape=shape)
+    y = paddle.static.data(name="y", shape=shape)
+    relu = paddle.nn.ReLU()
+    sigmoid = paddle.nn.Sigmoid()
+    y = sigmoid(relu(x))
+    
+    exe = paddle.static.Executor(paddle.CPUPlace())
+    exe.run(paddle.static.default_startup_program())
+
+    # convert to OpenVINO IR format
+    ov_model = convert_model(exe, example_input=[x], example_output=[y])
+
+    # optional: serialize OpenVINO IR to *.xml & *.bin
+    from openvino.runtime import serialize
+    serialize(ov_model, "ov_model.xml", "ov_model.bin")
```

Comment:
useless example. We need to start already pre-defined model, no need to demonstrate how to create PDPD model from scratch

Reply:
> useless example. We need to start already pre-defined model, no need to demonstrate how to create PDPD model from scratch

You know, in order to convert model successfully, we may pass specific parameters for different models. So, this example is to tell user how to convert PDPD model when facing `paddle.fluid.executor.Executor` property instead of demonstrate how to create PDPD model.